vagga.yaml

```diff
@@ -100,12 +111,12 @@ commands:
       app: *run-app
       db: *run-db
 
-  test: !Command
-    description: Run tests
-    container: test
+  lint: !Command
+    description: Run linters
+    container: lint
     run: pre-commit run --all-files
 
-  pre-commit: !Command
-    description: Run pre-commit
+  test: !Command
+    description: Run tests
     container: test
-    run: [pre-commit]
+    run: python3 manage.py test --pattern="*_test.py"
```

Comment:
Have you tried configuring it this way?
https://github.com/kpi-web-guild/django-girls-blog-DrEdi/blob/master/.pre-commit-config.yaml#L16